tools/manifest/download.py

```diff
@@ -62,13 +76,15 @@ def github_url(tags):
             logger.warning("Response was not valid JSON")
             return None
 
+        candidates = []
         for item in release["assets"]:
-            # Accept both ways of naming the manfest asset, even though
-            # there's no longer a reason to include the commit sha.
-            if item["name"].startswith("MANIFEST-") and item["name"].endswith(".json.gz"):
-                return item["browser_download_url"]
-            elif item["name"] == "MANIFEST.json.gz":
-                return item["browser_download_url"]
+            score = score_name(item["name"])
+            if score is not None:
+                candidates.append((score, item["browser_download_url"]))
+
+        if candidates:
+            _, winner = sorted(candidates)[0]
```

Comment:
I think I would redesign this so we return the sorted list of all manifests, so that we can fall back to another format if we download a corrupt file or something.